docs/asciidoc/advancedFeatures.adoc

```diff
@@ -815,15 +817,24 @@ completely different instance, although it is really not recommended (doing so m
 impact third-party libraries that depend on the `Context`).
 
 [[context.read]]
-=== Reading a `Context`
+=== Reading a `Context`, through the `ContextView`
 
-Once you haved populated a `Context`, you can retrieve that data.
+Once you have populated a `Context`, you may want to peek into it at runtime.
 Most of the time, the responsibility of putting information into the `Context`
 is on the end user's side, while exploiting that information is on the third-party library's side,
 as such libraries are usually upstream of the client code.
 
-The tool for reading data from the context is the static `Mono.subscriberContext()`
-method.
+The read oriented operators allow to obtain data from the `Context` in a chain of operators by exposing
+its `ContextView`:
+
+ - to access the context from a source-like operator, use `deferWithContext` factory method
+ - to access the context from data signals, use `doOnEach` and `Signal#getContext()`
```

Comment:
so we're not deprecating `Signal#getContext`?

Reply:
we're not deprecating `Signal.getContext` but we'll stop advertising it in the reference guide